docs/getting-started/example-deployment/using-the-api.md

```diff
@@ -0,0 +1,191 @@
+# Using the API
+
+!!! info "Work in Progress"
+
+    This page is under construction.
+
+As covered in the [Gaffer basics](../basics.md) the main POST request used in the API is
+`/graph/operations/execute`. This part of the guide will cover the general usage of this part of the
+API and walk through some general operations you might want to use to load data or query.
+
+!!! note
+    The [Gaffer operations reference guide](../../reference/operations-guide/operations.md)
+    has more detail on available operations.
+
+## Loading Data
+
+Gaffer supports various aspects of loading data, generally this is done in a few stages which are
+outlined in the following diagram.
```

Comment:
Loading data in production is not really done using the REST API with generate elements. There are some other loading methods used which have varying degrees of documentation or direct loading into Accumulo which bypasses Gaffer.
For this example it's enough to say e.g: for small amounts of data the following steps can be used.